src/test/java/pl/touk/sputnik/processor/codenarc/CodeNarcReviewProcessorFactoryTest.java

```diff
@@ -1,34 +1,33 @@
 package pl.touk.sputnik.processor.codenarc;
 
+import org.junit.jupiter.api.Test;
 import pl.touk.sputnik.configuration.Configuration;
 import pl.touk.sputnik.configuration.GeneralOption;
 import pl.touk.sputnik.processor.ReviewProcessorFactory;
 
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-import static org.mockito.Matchers.eq;
+import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
+import static pl.touk.sputnik.SputnikAssertions.assertThat;
 
-public class CodeNarcReviewProcessorFactoryTest {
+class CodeNarcReviewProcessorFactoryTest {
 
     @Test
-    public void testIsEnabled() throws Exception {
+    void testIsEnabled() {
         Configuration configuration = mock(Configuration.class);
         when(configuration.getProperty(eq(GeneralOption.CODE_NARC_ENABLED))).thenReturn("true");
 
         ReviewProcessorFactory factory = new CodeNarcReviewProcessorFactory();
-        assertTrue(factory.isEnabled(configuration));
+        assertThat(factory.isEnabled(configuration)).isTrue();
     }
 
     @Test
-    public void testCreate() throws Exception {
+    void testCreate() {
         Configuration configuration = mock(Configuration.class);
         when(configuration.getProperty(eq(GeneralOption.CODE_NARC_ENABLED))).thenReturn("true");
 
         ReviewProcessorFactory factory = new CodeNarcReviewProcessorFactory();
-        assertNotNull(factory.create(configuration));
+        assertThat(factory.create(configuration)).isNotNull();
```

Comment:
is it really readable?

Reply:
I think it is. I prefer assertj over junit assertions.